src/blockchain/Blockchain.ts

```diff
@@ -37,7 +37,7 @@ export interface IBlockchainApi {
   api: ApiPromise
 
   getStats(): Promise<Stats>
-  listenToBlocks(listener: (header: Header) => void): Promise<any> // TODO: change any to something meaningful
+  listenToBlocks(listener: (header: Header) => void): SubscriptionResult 
```

Comment:
The subscriptionResult is a type given from the Polkadot API, I don't think it actually suits here. 

I will look at the type here again.